packages/slice-machine/src/features/customTypes/customTypesBuilder/PageSnippetDialog/PageSnippetDialog.tsx

```diff
@@ -27,11 +31,26 @@ const PageSnippetContent: FC<PageSnippetContentProps> = ({ model }) => {
     return null;
   }
 
+  const trackOpenSnippet = () => {
+    void telemetry.track({
+      event: "page-type:open-snippet",
+      ...(config !== undefined
```

Comment:
Ok I see here why you have a framework that can be undefined. But actually it's not possible, for me, you can just add the condition above.

Like:

```suggestion
     if (config !== undefined) {
       // track
     }
```